app/controllers/experience_stages_controller.rb

```diff
@@ -5,6 +5,7 @@ def show
     @experience_stage = @challenge.experience_stage
     @themes = @experience_stage.themes
     @featured_experiences = @experience_stage.experiences.where(featured: true)
+    @order_field = params[:order_by].blank? ? '' : 'created_at DESC'
```

Comment:
I'd want to do a bit more validation on the scoping that we're going to use in the view here – I'm also wondering how hard it would be to add `direction` param so that we could toggle between ascending and descending as well.